test/api.js

```diff
@@ -42,7 +42,11 @@ module.exports = function api(smocks) {
     label: 'App stats',
     path: `${BASE_URL}/apps/{guid}/stats`,
     handler: function (req, reply) {
-      reply(appStats);
+      var app = apps.filter((a) => a.guid === req.params.guid).pop();
+      var response = Object.assign({}, appStats, {
+        state: app.state
```

Comment:
Yup, you're right. Looks like it returns the following `400` error:

```
{
  "code": 200003,
  "description": "Could not fetch stats for stopped app: help",
  "error_code": "CF-AppStoppedStatsError"
}
```

I'll fix that, nice catch